decidim-proposals/lib/decidim/proposals/proposal_serializer.rb

```diff
@@ -71,6 +79,15 @@ def related_proposals
       def url
         Decidim::ResourceLocatorPresenter.new(proposal).url
       end
+
+      def user_endorsements
+        proposal.endorsements.for_listing.map { |identity| identity.normalized_author&.name }
+      end
+
+      def original_proposal_url
+        return unless proposal.emendation?
```

Comment:
Shouldn't we check that `proposal.amendable` is not nil?

Reply:
You are right! Done!